### PR TITLE
Ui dim

### PR DESCRIPTION
**Pull request by heartjigi**
make dimension flexible on each device.
Almost "fixed value of dim" change dynamic

### CODE REVIEW

File: src/components/OnBoarding/OnBoardingPage.js [hunk 1]
```diff
@@ -69,32 +79,64 @@ class OnBoardingPage extends Component {
...
+        height: dimensions.fontWeight * 7,
+        borderRadius: dimensions.fontWeight * 4,
+        marginLeft: dimensions.widthWeight * 6,
+        marginRight: dimensions.widthWeight * 6,
+        marginTop: dimensions.heightWeight * 3,
       }} />
     );
   }
 
+  controlScroll(e, state) {
```
Comment (SarahhhC) [inline src/components/OnBoarding/OnBoardingPage.js]: What is `e`?

File: src/components/Shared/Dimensions.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Dimensions } from 'react-native';
+
+export const refDimensions = {
+  width: 375,
+  height: 667,
+};
+export const dimensions = {
+  width: Dimensions.get('window').width,
```
Comment (SarahhhC) [inline src/components/Shared/Dimensions.js]: Why don't you use `Dimensions.get('window').width` or other things as variable?

File: src/utils/UserUtil.js [hunk 3]
```diff
@@ -13,7 +13,8 @@ class UserUtil {
 
   // Sign In with Facebook.
   signInWithFacebook(callback) {
-    this.signOut();
+    console.log('signInWithFacebook');
```
Comment (SarahhhC) [inline src/utils/UserUtil.js]: delete console.

**Review by SarahhhC — COMMENTED** [review]
Please check comments!

File: src/components/OnBoarding/OnBoardingPage.js (hunk 1 above)
Comment (heartjigi) [reply src/components/OnBoarding/OnBoardingPage.js]: e is event,
This is original case of onMomentumScrollEnd for Scrollview,
for get state, e is needed maybe..

File: src/components/Shared/Dimensions.js (hunk 2 above)
Comment (heartjigi) [reply src/components/Shared/Dimensions.js]: Sry I can't understand your intend,
Is that not variable?

File: src/utils/UserUtil.js (hunk 3 above)
Comment (heartjigi) [reply src/utils/UserUtil.js]: Oh... eagle eye.. Thanks

File: src/components/OnBoarding/OnBoardingPage.js (hunk 1 above)
Comment (SarahhhC) [reply src/components/OnBoarding/OnBoardingPage.js]: Can't it be `event` instead `e`?

Comment (heartjigi) [reply src/components/OnBoarding/OnBoardingPage.js]: solved

**Comment by ifndefdeadmau5** [comment]
LGTM Very good

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/139/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**